pkg/frontend/querymiddleware/codec_test.go

```diff
@@ -111,6 +111,130 @@ func TestMetricsQueryRequest(t *testing.T) {
 	}
 }
 
+func TestMetricsQuery_MinMaxTime(t *testing.T) {
+
+	rangeVectorDurationStr := "5m"
+	rangeVectorDuration, _ := time.ParseDuration(rangeVectorDurationStr)
+	rangeVectorDurationMS := rangeVectorDuration.Milliseconds()
+
+	offsetDurationStr := "1h"
+	offsetDuration, _ := time.ParseDuration(offsetDurationStr)
+	offsetDurationMS := offsetDuration.Milliseconds()
+
+	for _, testCase := range []struct {
+		name         string
+		metricsQuery MetricsQueryRequest
+		expectedMinT int64
+		expectedMaxT int64
+		expectedErr  error
```

Comment:
`expectedErr` is always `nil` - is this needed?

Or are there unhappy path test cases missing?